### ISSUE TITLE
No more F8 variant?

### ISSUE THREAD

**Issue by BEST8OY** [Artoriuz/ArtCNN#35]
Hello
I asked [here](https://github.com/Artoriuz/ArtCNN/issues/15) but I didn't get any answer!
The F8 variant is a really nice shader for middle range laptops!
Are you going to not maintain that variant anymore?

**Comment by Artoriuz** [comment]
Yes, I don't have any intention to bring C4F8 back. If C4F16 is too slow just use something like RAVU instead.
